e2e/cypress/integration/ui/settings/04_project_mgmt.spec.ts

```diff
@@ -0,0 +1,206 @@
+interface Project {
+  id: string;
+  name: string;
+  type: string;
+}
+
+// assume 2.0 if not in CI. if you wish something different start cypress with
+// IAM_VERSION set to what you are testing.
+if (Cypress.env('IAM_VERSION') === undefined) {
```

Comment:
Can we put this logic, and the `describeIAMV2P1` method, into some helpers?

Reply:
Unfortunately it's not possible to run custom Cypress commands outside running tests. It would require adding an extra `describe` and `it` block to the tests that use this helper, which seems to me like it would defeat the purpose of the helper. I'll investigate how to add `utility` helpers that don't require the Cypress test runner to be up